.clang-tidy

```diff
@@ -39,6 +39,11 @@ Checks: >
   readability-redundant-smartptr-get,
   readability-redundant-string-cstr
 
+Checks: '-modernize-use-using'
```

Comment:
can the `Checks` field be repeated?

Reply:
good q. will take a look tomorrow!

Reply:
probably this line isnt necessary - `modernize-use-using` is already listed i think

Reply:
(or just needs to be removed above perhaps)